qa/suites/rgw/iam/tasks/iamtests.yaml

```diff
@@ -0,0 +1,8 @@
+tasks:
+- s3tests:
+    client.0:
+      iam_tests: True
+      extra_attrs: ["test_of_iam","!fails_on_rgw"]
+      force-branch: master
+      git_remote: https://github.com/Seagate/
```

Comment:
Is this section going to be removed?

Reply:
yes, I have left it here for the time being, so that if you have any comment on the PR, I can re-run it through teuthology.